pkgs/development/python-modules/py-deprecate/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, scikit-learn
+, pytorch
+}:
+
+let
+  pname = "pyDeprecate";
+  version = "0.3.2";
+in
+buildPythonPackage {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "Borda";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-84RmQvLxwtLPQk3hX7Q6eeJeejhrO3t+mc95W1E85Fg=";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+    scikit-learn
+  ];
+
+  meta = with lib; {
+    description = "A python module for marking deprecated functions or classes and re-routing to the new successors' instance. Used by torchmetrics";
```

Comment:
I think we should strip the Python Module part. That obvious from the package name.